deps/FFmpeg/dotnet/build.sh

```diff
@@ -168,6 +236,8 @@ NUMBER_OF_CORES=$(nproc)
 
 # Uncomment a line to run that configuration
 #build_windows_x64_shared
-build_windows_x86_shared
+#build_windows_x86_shared
```

Comment:
Is x86 disabled on purpose?

Reply:
Will revert that one - in general all those custom build scripts need to be taken with a grain of salt. In their current state you can use them as some sort of reference point, but none of them were working out of the box. Either wrong paths or out-dated ones, wrong output paths, out-commented code, etc.